src/content/content.js

```diff
@@ -1,12 +1,35 @@
 import React from 'react';
 import ReactDom from 'react-dom';
+import { ComponentName, ListComponentName } from '../compName';
 import './content.scss';
 
-export const Content = () =>{
-    const text = 'content'
+
+export const Content = () => {
+    
     return (
         <div className="content">
-            <p>{text}</p>
+            
+            <ListComponentName 
+                users={[
+                
+                    {
+                        firstName:'Petia',
+                        lastName :'Petrov',
+                        age :'25' 
+                    }
+                    ,
+                    {
+                        firstName:'Petia',
+                        lastName :'Petrov',
+                        age :'25' 
+                    },
+                    {
+                        firstName:'Petia',
+                        lastName :'Petrov',
+                        age :'25' 
+                    }
+                ]} 
```

Comment:
it is better do not to use data directly in components. Better way - to crete variable for the users array and pss it to component `users={users}`